src/server/graphql/models/Meeting/meetingMutation.js

```diff
@@ -1,20 +1,20 @@
 import {Meeting} from './meetingSchema';
-import r from '../../../database/rethinkdriver';
-import {isLoggedIn} from '../authorization';
+import r from '../../../database/rethinkdriver'; // eslint-disable-line id-length
+// import {isLoggedIn} from '../authorization';
```

Comment:
yeah, we can go ahead & nuke all the auth stuff. since falcor was auth-free i tried to stay in-scope for the port to graphql, but i left the tools there so when the times comes we don't have to think too hard about where to put it.